client/src/modules/stock/entry/modals/lots.modal.html

```diff
@@ -4,17 +4,23 @@
   novalidate>
 
   <div class="modal-header">
-    <div class="headercrumb">
-      <i class="fa fa-tags"></i> 
-      <h3 style="margin: 0px;">{{ $ctrl.inventory.inventory.label }}</h3>
-    </div>
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.STOCK</li>
+      <li class="static" translate>STOCK.ENTRY</li>
+      <li class="title">{{ $ctrl.inventory.inventory.label }}</li>
+    </ol>
   </div>
 
   <div class="modal-body clearfix">
     <div class="form-group">
-      <label translate>FORM.LABELS.GLOBAL_QUANTITY</label>
+      <label class="control-label" translate>FORM.LABELS.GLOBAL_QUANTITY</label>
+      
+      <div ng-if="$ctrl.entryType === 'purchase' || $ctrl.entryType === 'transfer_reception'">
+        <span class="form-control-static">{{ $ctrl.inventory.quantity }}</span>
+      </div>
+
       <input 
-        ng-disabled="$ctrl.entryType === 'purchase' || $ctrl.entryType === 'transfer_reception'" 
+        ng-if="$ctrl.entryType !== 'purchase' && $ctrl.entryType !== 'transfer_reception'" 
```

Comment:
Instead of repeating this check multiple times, a better way to express this would be in JS.  Something like this:
```js
// determine if the inventory quantity and cost should be editable or not.
$ctrl.hasEditableInventory = ($ctrl.entryType !== 'purchase' && $ctrl.entryType !== 'transfer_reception');
```
... later in the HTML...
```html
<div ng-if="$ctrl.hasEditableInventory">
   <!--- put all of the editable code here -->
</div>
<div ng-if="!$ctrl.hasEditableInventory">
  <!-- put all form-static code here -->
</div>
```